Resources/Maps/saltern.yml

```diff
@@ -47599,4 +47460,722 @@ entities:
+    pos: -7.5,-29.5
+    rot: 3.141592653589793 rad
+    type: Transform
+- uid: 4255
+  type: KitchenMicrowave
+  components:
+  - parent: 855
+    pos: -7.5,-28.5
+    rot: 3.141592653589793 rad
+    type: Transform
+  - containers:
+      microwave_entity_container:
+        type: Robust.Server.GameObjects.Components.Container.Container
+    type: ContainerContainer
+- uid: 4256
+  type: ClothingHeadHelmetScaf
+  components:
+  - parent: 855
+    pos: 13.5,24.5
+    rot: 3.141592653589793 rad
+    type: Transform
+- uid: 4257
+  type: ClothingBeltSecurityFilled
+  components:
+  - parent: 855
+    pos: -6.5,13.5
+    rot: 3.141592653589793 rad
+    type: Transform
+  - containers:
+      storagebase:
+        type: Robust.Server.GameObjects.Components.Container.Container
+    type: ContainerContainer
+- uid: 4258
+  type: Shelf
+  components:
+  - parent: 855
+    pos: -20.5,7.5
+    rot: 3.141592653589793 rad
+    type: Transform
+- uid: 4259
+  type: SprayBottleSpaceCleaner
+  components:
+  - parent: 855
+    pos: -20.5,7.5
+    rot: 3.141592653589793 rad
+    type: Transform
+- uid: 4260
+  type: RCDAmmo
+  components:
+  - parent: 855
+    pos: 32.5,-2.5
+    rot: 3.141592653589793 rad
+    type: Transform
+- uid: 4261
+  type: RCDAmmo
+  components:
+  - parent: 855
+    pos: 32.812416,-2.3740568
+    rot: 3.141592653589793 rad
+    type: Transform
+- uid: 4262
+  type: SmgWt550
+  components:
+  - parent: 855
+    pos: -8.457222,21.596565
+    rot: 3.141592653589793 rad
+    type: Transform
+  - maxAngle: 59.99999999999999
+    type: MagazineBarrel
+  - containers:
+      MagazineBarrel-chamber:
+        type: Content.Server.GameObjects.ContainerSlot
+      MagazineBarrel-magazine:
+        entities:
+        - 4263
+        type: Content.Server.GameObjects.ContainerSlot
+    type: ContainerContainer
+- uid: 4263
+  type: MagazinePistolSmgTopMounted
+  components:
+  - parent: 4262
+    type: Transform
+  - containers:
+      RangedMagazine-magazine:
+        type: Robust.Server.GameObjects.Components.Container.Container
+    type: ContainerContainer
+- uid: 4264
+  type: BodyBag_Container
+  components:
+  - parent: 855
+    pos: 14.594889,-13.278449
+    rot: 3.141592653589793 rad
+    type: Transform
+  - containers:
+      EntityStorageComponent:
+        type: Robust.Server.GameObjects.Components.Container.Container
+      body_bag_label:
+        type: Content.Server.GameObjects.ContainerSlot
+    type: ContainerContainer
+- uid: 4265
+  type: BodyBag_Container
+  components:
+  - parent: 855
+    pos: 15.594889,-13.262824
+    rot: 3.141592653589793 rad
+    type: Transform
+  - containers:
+      EntityStorageComponent:
+        type: Robust.Server.GameObjects.Components.Container.Container
+      body_bag_label:
+        type: Content.Server.GameObjects.ContainerSlot
+    type: ContainerContainer
+- uid: 4266
+  type: FoodMemoryLeek
+  components:
+  - parent: 855
+    pos: -10.516438,0.52631223
+    type: Transform
```

Comment:
IMO this shouldn't be in bar, especially when it can mess with reagents.

Reply:
Agreed.